### ISSUE TITLE
Research legal resources for nonprofits in specific state

### ISSUE THREAD

**Issue by avidrucker** [knowledge-for-good/knowledgeforgood#131]

**Comment by avidrucker** [comment]
@seekanddefine We already have some research on this, correct? Also, what are our specific research goals? Perhaps we can get some guidance on this from a "subject matter expert".

**Comment by avidrucker** [comment]
Note, this is not blocked by #127 ... but it may be pragmatic to decide before doing X~ hours of research.

**Comment by avidrucker** [comment]
@caryfado Would you be into helping us create one or more SMART goals for this? I am going to split this into two issues (one for NY, and one for Virginia), as well.
